freemocap/core_processes/process_motion_capture_videos/process_session_folder.py

```diff
@@ -36,7 +36,9 @@
     test_mediapipe_2d_data_shape,
 )
 from freemocap.tests.test_mediapipe_3d_data_shape import test_mediapipe_3d_data_shape
+from freemocap.utilities.flatten_3d_data import flatten_3d_data
```

Comment:
should change the name of this to something like "project to Y plane" or something

The term "flatten 3d data" is ambiguous because it shadows the name of a commonly used function `numpy.flatten` which flattend higher dimensional numpy arrays into 2d arrays.